content/architecture/application/altinn-platform/receipt/_index.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Application architecture receipt component - Altinn Platform
+linktitle: Receipt
+description: Description of receipt component
+tags: [architecture, solution, receipt]
+weight: 100
+---
+
+The profile component is an .NET Core MVC Application exposing a react receipt application, and an internal api for getting the current user.
+
+The receipt frontend view is available at `http://at21.platform.altinn.cloud/receipt/{instanceOwnerId}/{instanceId}`,
+example [https://platform.at21.altinn.cloud/receipt/50002121/a78b3029-009b-4fbc-995d-d8d2abf6f74b](https://platform.at21.altinn.cloud/receipt/50002121/a78b3029-009b-4fbc-995d-d8d2abf6f74b)
+The receipt component also exposes an api which is available at `http://at21.platform.altinn.cloud/receipt/api/v1`, where available resources are defined below. This api is intended for
+use by the receipt frontend, if you need information about the user you should [use the profile component.](/architecture/application/altinn-platform/profile)
+
+To use the receipt component you need to have a valid `AltinnStudioRuntime` cookie. To obtain this you need to:
```

Comment:
This information seems more relevant for developers than external people comming to find information. E.g. no one outside the organization has information about the test users and passwords to log into at21.